react/features/participants-pane/reducer.js

```diff
@@ -0,0 +1,36 @@
+import { ReducerRegistry } from '../base/redux';
+
+import {
+    PARTICIPANTS_PANE_CLOSE,
+    PARTICIPANTS_PANE_OPEN,
+    REDUCER_KEY
+} from './constants';
+
+const DEFAULT_STATE = {
+    isOpen: false
+};
+
+/**
+ * Listen for actions that mutate the participants pane state
+ */
+ReducerRegistry.register(
+    REDUCER_KEY, (state = DEFAULT_STATE, action) => {
+        switch (action.type) {
+
+        case PARTICIPANTS_PANE_CLOSE:
```

Comment:
Wouldn't it be better to have a single action which takes a parameter?

Reply:
I usually prefer being explicit but I can merge them if you like it better that way.